src/components/header/RadioSearchInput.jsx

```diff
@@ -8,27 +8,25 @@ const RadioSearchInput = () => {
   const [isInputClicked, setIsInputClicked] = useState(false);
 
   return (
-    <div className="mt-10 ml-5 flex w-[348px] items-center gap-2">
-      <div className="flex h-11 flex-1 items-center rounded-4xl bg-neutral-200 px-4">
-        <SearchThinIcon className="mr-2" />
-        <input
-          onFocus={() => {
-            setIsInputClicked(true);
-          }}
-          onBlur={() => {
-            setIsInputClicked(false);
-          }}
-          type="text"
-          placeholder={isInputClicked ? " " : "Search by radio name"}
-          className="w-full text-sm font-semibold text-neutral-800 outline-none"
-        />
+    <div className="flex w-full justify-center pt-10">
+      <div className="flex w-full max-w-md items-center gap-2 px-4">
+        <div className="flex h-11 flex-1 items-center rounded-4xl bg-neutral-200 px-4">
+          <SearchThinIcon className="mr-2" />
+          <input
+            onFocus={() => setIsInputClicked(true)}
+            onBlur={() => setIsInputClicked(false)}
+            type="text"
+            placeholder={isInputClicked ? " " : "Search by radio name"}
```

Comment:
```suggestion
            placeholder="Search by radio name"
```
input의 placeholder는 포커스되면 기본적으로 사라지기 때문에 isInputClicked로 빈 문자열 처리하지 않아도 될 것 같아요~
삼항 연산자 없이 "Search by radio name"만 써도 동작엔 문제가 없어요~